ansible/roles/kong-api/defaults/main.yml

```diff
@@ -6552,3 +6554,617 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+
+# UCI Related
+  - name: getUserSegmentsUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/userSegment/get"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/userSegment/get"
```

Comment:
Missing variable `uci_admin_service_url` in this definition file. This will result in ansible errors